window.go

```diff
@@ -54,8 +56,10 @@ const (
 	EventNameWindowEventUnresponsive                  = "window.event.unresponsive"
 	EventNameWindowEventDidGetRedirectRequest         = "window.event.did.get.redirect.request"
 	EventNameWindowEventWebContentsExecutedJavaScript = "window.event.web.contents.executed.javascript"
+	EventNameWindowEventWebContentsSetProxy           = "window.event.web.contents.set.proxy"
 	EventNameWindowEventWillNavigate                  = "window.event.will.navigate"
 	EventNameWindowEventUpdatedCustomOptions          = "window.event.updated.custom.options"
+	EventNameWindowLoadedURL                          = "window.event.loaded.url"
```

Comment:
Name should be `EventNameWindowEventLoadedURL` (with `Event` in the middle)

Also could you move this field upwards so that fields are sorted alphabetically?